source/_components/homematicip_cloud.markdown

```diff
@@ -11,6 +11,7 @@ logo: homematicip_cloud.png
 ha_category: Hub
 ha_iot_class: "Cloud Push"
 ha_release: 0.66
+last update: 0.87.0
```

Comment:
this option did not exist, so please omit it.

Reply:
Ok - Done